src/components/new-request/search.vue

```diff
@@ -45,9 +45,42 @@
         />
       </v-col>
 
+      <!-- Business Lookup/Fetch -->
+      <v-col v-if="showBusinessLookup" cols="12" md="6" class="business-lookup">
+        <template v-if="!business">
+          <BusinessLookup v-if="getIsAuthenticated" @business="onBusiness($event)"/>
+          <BusinessFetch v-else @business="onBusiness($event)"/>
+        </template>
+        <v-text-field
+          v-else
+          append-icon="mdi-close"
+          readonly
+          filled
+          :label="business.identifier"
+          :value="business.legalName"
+          @click:append="onBusiness(null)"
+          @keyup.delete="onBusiness(null)"
+          :rules="rules"
+          autofocus
+          :hint="getIsAuthenticated ?
+           'Search by name, incorporation or registration number of existing business' :
+            'Enter registration number of existing business'"
+          persistent-hint
+        />
+      </v-col>
+
       <!-- Entity Type -->
-      <v-col v-if="showEntityType" cols="12" :md="getIsXproFlow ? 4 : 6">
+      <v-col
+        v-if="showEntityType" cols="12"
+        :md="getIsXproFlow ? 4 : ( isConversion ? 12 : 6)">
+        <v-text-field
+          v-if="isConversion && isBenBusiness"
```

Comment:
The logic on lines 74 and 77 is getting complicated (nested ifs). (Also line 75.)

Can you lay out the logic cleaner? I'm thinking maybe using `<template v-if="...">` or separate columns for BEN conversion vs others?

Reply:
This will be handled in bcgov/entity#17608.